e2e-tests/cypress/tests/cypress.config.ts

```diff
@@ -30,6 +41,7 @@ export default defineConfig({
   e2e: {
     specPattern: "**/*.feature",
     excludeSpecPattern: "*.js",
+    experimentalSessionAndOrigin: true,
     baseUrl: "http://localhost:3000",
```

Comment:
```suggestion
    baseUrl: 'http://localhost:3000',
```

Reply:
Sure, this will be handled in a seperate task to adapt the linting to our repo's standards.